tests/components/deconz/test_sensor.py

```diff
@@ -1074,9 +1070,8 @@ async def test_allow_clip_sensors(
     assert hass.states.get("sensor.clip_flur").state == "0"
 
 
-async def test_add_new_sensor(
-    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker, mock_deconz_websocket
-) -> None:
+@pytest.mark.usefixtures("config_entry_setup")
+async def test_add_new_sensor(hass: HomeAssistant, mock_deconz_websocket) -> None:
```

Comment:
https://github.com/home-assistant/core/pull/120967#discussion_r1665452557

Reply:
Will do in a separate PR